crates/database/src/auction_participants.rs

```diff
@@ -0,0 +1,55 @@
+use {
+    crate::{auction::AuctionId, Address},
+    sqlx::PgConnection,
+};
+
+/// Participants of a solver competition for a given auction.
+#[derive(Debug, Clone, PartialEq, sqlx::FromRow)]
+pub struct Participants {
+    pub auction_id: AuctionId,
+    pub participants: Vec<Address>,
+}
+
+pub async fn insert(ex: &mut PgConnection, data: Participants) -> Result<(), sqlx::Error> {
+    const QUERY: &str =
+        r#"INSERT INTO auction_participants (auction_id, participants) VALUES ($1, $2);"#;
```

Comment:
I don't think participants should be an array here.

Reply:
Fixed.